core/jazz_create-serverless-service/test/test.js

```diff
@@ -173,6 +174,26 @@ describe('create-serverless-service', function () {
       configstub.restore();
     });
 
+    it("should inform user of error if given an event with no body.deployment_accounts", function () {
+      let configstub = sinon.stub(configModule, "getConfig").returns({
+        "DEPLOYMENT_ACCOUNTS": [{"accountId":"12345","region":"us-east-1","provider":"aws","primary":true},{"accountId":"67890","region":"us-west-2","provider":"aws","primary":false}],
+        "DEPLOYMENT_TARGETS": { "gcp": "apigee", "function": "aws_lambda" }
+      });
+
+      event.body.deployment_accounts = null;
+      let errMessage = "Deployment accounts is missing or is not in a valid format";
+      let errType = "BadRequest";
+      index.handler(event, context, (err, res)=>{
+        console.log('err',err);
```

Comment:
remove console logs and commented lines. Please add assertions. Please also add test case to check if no primary account was supplied

Reply:
removed consoles, commented lines. Added assertions. Added new test cases which increased code coverage